### PR TITLE
Add visibility to properties and functions

### PR DESCRIPTION
**Pull request by sveneld**
Remove _ and @access as representment of function/property visibility

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php-version: ['5.6', '7.0', '7.1', '7.2', '7.3', '7.4']
+        php-version: ['5.6', '7.0', '7.1', '7.2', '7.3', '7.4', '8.0', '8.1']
```
Comment (jparise) [inline .github/workflows/ci.yml:17]: I just added all of the current 8.x releases 16d0509.

**Review by jparise — COMMENTED** [review]
Thanks for making this improvement!

It _will_ break API compatibility with any existing custom Log handlers, however (names changing due to their `_` prefixes being removed). This primarily applies to the `Log` attributes now marked as `protected`.

So I think we should do one of these things:
1. Restore the `_` prefixes for those protected attributes to maintain API compatibility.
2. Bump the package's major version and `api` version in `package.xml`.

Which approach do you think is the best?

**Comment by sveneld** [comment]
My opinion is to update major version

**Comment by sveneld** [comment]
And if will change the major version maybe it's time to drop support of php5 and set minimal version as 7.4 or better as 8.0?

**Comment by jparise** [comment]
> And if the major version will change, maybe it's time to drop support of php5 and set minimal version as 7.4 or better as 8.0?

Yes, I agree. I'll merge this change as-is and can review follow-up PRs for those changes.

**Review by jparise — APPROVED** [review]

**Pull request merged**